docs/src/test-runners-csharp.md

```diff
@@ -154,24 +154,18 @@ CLI. See the following example:
 ```xml
 <?xml version="1.0" encoding="utf-8"?>
 <RunSettings>
-  <TestRunParameters>
-    <Parameter name="browser" value="chromium" />
-    <Parameter name="headless" value="false" />
-    <Parameter name="channel" value="msedge" />
-  </TestRunParameters>
+  <Playwright>
+    <Browser>chromium</Browser>
+    <LaunchOptions>
```

Comment:
The more a look at this the more it feels that LaunchOptions should be under Browser.